www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+
+## `src/env`
+
+Used for environment variable validation and type definitions - see [Environment Variables](/en/usage/env-variables).
+
+### `src/env/client.mjs`
+
+[See the client.mjs documentation](/en/usage/env-variables#servermjs--clientmjs)
+
+### `src/env/server.mjs`
+
+[See the server.mjs documentation](/en/usage/env-variables#servermjs--clientmjs)
+
+### `src/env/schema.mjs`
+
+[See the schema.mjs documentation](/en/usage/env-variables#schemamjs)
+
+## `src/pages`
+
+The `pages` folder contains all the pages of the Next.js application. The `index.tsx` file at the root directory of `/pages` is the homepage of the application. The `_app.tsx` file is used to wrap the application with providers. See [Next.js documentation](https://nextjs.org/docs/basic-features/pages) for more information.
+
+## `src/pages/api`
+
+The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file (with Prisma) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file (with Next-Auth) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
+
+<sub>(With NextAuth.js, tRPC or tRPC + Prisma)</sub>
+
+## `src/pages/api/auth`
+
+The `auth` folder contains the NextAuth.js authentication slug route.
+
+<sub>(With NextAuth.js)</sub>
+
+### `src/pages/api/auth/[...nextauth].ts`
+
+The `[[...nextauth]].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](/en/usage/next-auth) for more information.
+
+<sub>(with NextAuth.js)</sub>
+
+## `src/pages/api/trpc`
+
+The `trpc` folder contains the tRPC API entry point.
+
+<sub>(with tRPC)</sub>
+
+### `src/pages/api/trpc/[trpc].ts`
+
+The `[[trpc]].ts` file is the tRPC API entry point. It is used to handle tRPC requests. See [tRPC usage](en/usage/trpc#-pagesapitrpctrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `src/server`
+
+The `server` folder is used to clearly seperate server-side code from client-side code.
+
+<sub>(with tRPC and/or Prisma)</sub>
+
+## `src/server/common`
+
+The `common` folder contains commonly re-used server-side code.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/common/get-server-auth-session.ts`
+
+The `get-server-auth-session.ts` file is used to get the NextAuth.js session on the server-side. See [NextAuth.js usage](/en/usage/next-auth#usage-with-trpc) for more information.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+## `src/server/db`
+
+The `db` folder contains the Prisma client.
+
+<sub>(with Prisma)</sub>
+
+### `src/server/db/client.ts`
+
+The `client.ts` file is used to instantiate the Prisma client at global scope. See [Prisma usage](/en/usage/prisma#prisma-client) for more information.
```

Comment:
```suggestion
The `client.ts` file is used to instantiate the Prisma client at global scope. See [Prisma usage](/usage/prisma#prisma-client) for more information.
```

Reply:
Fixed in https://github.com/t3-oss/create-t3-app/commit/9a4ac76852359e187ea183f52165628b64e0d26d